lib/utils.js

```diff
@@ -868,13 +880,107 @@ exports = {
                              resource.replace("/", "_"));
         if (fs.existsSync(filename)) {
           return filename;
-        } else {
-          return null;
         }
+        return null;
       } catch (err) { console.log("Stored file error:", err);}
```

Comment:
hmm, i know this was already here, but it's not normal to log in libraries... if we do it elsewhere, it's okay here too.